tests/test_ops.py

```diff
@@ -427,6 +427,166 @@ def test_overlap():
     )
     assert len(b) == 3
 
+    ### test overlap with point and segment data
```

Comment:
isn't this now the same as  the test 
### Point adjacent to the start of the segment should
    ### overlap with the segment
?
can one of these be removed @smitkadvani  ?

Reply:
In one test, point interval(['chr1', 2, 2]) is adjacent to open ended side of segment interval point(['chr1', 1, 2]) and for the other one point interval(['chr1', 1, 1]) is adjacent to closed ended side of segment interval point(['chr1', 1, 2])